packages/jest-preset-mc-app/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@commercetools-frontend/jest-preset-mc-app",
-  "version": "16.12.0",
+  "version": "16.12.1",
```

Comment:
Please don't chagne this. Bots/automation does :)

Reply:
That's exactly who did it, not me :D